resources/fetching/fetcher.go

```diff
@@ -69,6 +69,14 @@ const (
 	KeyManagement          = "key-management"
 	ProjectManagement      = "project-management"
 	DataProcessing         = "data-processing"
+
+	// Azure resource types
+	AzureVMType               = "azure-vm"
+	AzureStorageAccountType   = "azure-storage-account"
+	AzureNetworkInterface     = "azure-network-interface"
+	AzureApplicationService   = "azure-application-service"
+	AzureLoggingAndMonitoring = "azure-logging-and-monitoring"
+	AzureDatabaseService      = "azure-database-service"
```

Comment:
```suggestion
```

Let's add these one by one as we implement the rules. We can merge this PR without them.